libs/components/config/src/lib/params.ts

```diff
@@ -226,7 +226,9 @@ export class SkyAppRuntimeConfigParams {
     // Add provided parameters to the URL.
     if (queryParams) {
       for (const [paramName, decodedValue] of Object.entries(queryParams)) {
-        unifiedParams = unifiedParams.set(paramName, decodedValue);
+        if (decodedValue) {
```

Comment:
Angular's `queryParams` supports empty string values (see the test above), so we should probably make this check more specific:
```
if (decodedValue !== undefined) {
```

Reply:
Updated